instana/util/ids.py

```diff
@@ -35,6 +36,35 @@ def generate_id():
     return new_id
 
 
+def header_to_id_no_truncation(header):
```

Comment:
Since this function is to stay and `header_to_id` is kept for backwards compatibility, you might consider to name it to reflect the expected result instead of behaviour:
```suggestion
def header_to_long_id(header):
```